### PR TITLE
[MRG] ENH:  Set initial time in Brain.add_data() 

### PR DESCRIPTION
**Pull request by christianbrodbeck**
Following up on https://github.com/mne-tools/mne-python/pull/3375 CC @jaeilepp @Eric89GXL

### CODE REVIEW

**Comment by agramfort** [comment]
LGTM

+1 for merge if you're done.

**Comment by christianbrodbeck** [comment]
Updated the example. If others agree with the API then this is ready.

**Comment by agramfort** [comment]
LGTM

File: surfer/viz.py [hunk 1]
```diff
@@ -857,6 +858,8 @@ def add_data(self, array, min=None, max=None, thresh=None,
             conserving memory when displaying different data in a loop.
         time_label_size : int
             Font size of the time label (default 14)
+        initial_time : float
```
Comment (Eric89GXL) [inline surfer/viz.py]: `| None`
and describe it means the first time point will be used

**Comment by Eric89GXL** [comment]
Otherwise LGTM, but you should probably also add a test that the parameter at least can be passed in

**Comment by christianbrodbeck** [comment]
Ready and tests pass, added test and doc.

File: surfer/tests/test_viz.py [hunk 2]
```diff
@@ -191,16 +191,26 @@ def test_meg_inverse():
     stc = io.read_stc(stc_fname)
     data = stc['data']
     vertices = stc['vertices']
-    time = 1e3 * np.linspace(stc['tmin'],
-                             stc['tmin'] + data.shape[1] * stc['tstep'],
-                             data.shape[1])
+    time = np.linspace(stc['tmin'], stc['tmin'] + data.shape[1] * stc['tstep'],
+                       data.shape[1])
```
Comment (Eric89GXL) [inline surfer/tests/test_viz.py]: The endpoint needs to be `stc['tmin'] + (data.shape[1] - 1) * stc['tstep']` right?

**Comment by Eric89GXL** [comment]
Otherwise LGTM

**Comment by christianbrodbeck** [comment]
Good catch @Eric89GXL

**Comment by Eric89GXL** [comment]
Thanks @christianbrodbeck

**Pull request merged**